src/borg/testsuite/efficient_bytes_queue.py

```diff
@@ -0,0 +1,40 @@
+from tempfile import TemporaryFile
+
+import pytest
+
+from ..helpers.datastruct import EfficientBytesQueue
+
+
+class TestEfficientQueue:
+    def test_base_usage(self):
+        sut = EfficientBytesQueue(100)
```

Comment:
i had to think a bit until i found that sut is "subject under test" likely. :-)

is this a frequent name in unit tests?

Reply:
Yes it means "subject under test". Frequent depends on where :). I use it quite often where I work, but I can adjust to whatever is your preference. Seemingly you don't use any convention so I can rename it `queue` if you don't like `sut`. I don't have hard preferences.

Reply:
yeah, queue might be easier.